### PR TITLE
fix #70507 unyenethaber.com

### LINKED ISSUE #70507: unyenethaber.com
### Issue URL (Ads)
 [http://www.unyenethaber.com/uzaktan-egitim-22-ocaka-kadar-surecek.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.unyenethaber.com%2Fuzaktan-egitim-22-ocaka-kadar-surecek.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.25
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#70507

### CODE REVIEW

**Comment by gork7777** [comment]
Hey dude, I'm new here. How are you doing commit into "master"?

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -389,6 +389,11 @@ selamtuzla.com##.aside_ad
 kibriswebhaber.com##img[width="1100"][height="140"]
 petroturk.com##.site-content > div[class^="site-"]
 petroturk.com##div[class^="ads-"]
+unyenethaber.com##.genel > div[style^="width: 980px; height: auto;"] > a
+unyenethaber.com##.haberDetayIcerik > div[style^="margin-right: 10px; float: left;"]
+unyenethaber.com###siteSol > div[style^="width: 780px; height: auto;"]
+unyenethaber.com###siteSag > div[style^="width: 370px; height: auto;"]
+unyenethaber.com###haberAlani > div[style^="width: 100%; height: auto; overflow: hidden;"]
```
Comment (AdamWr) [inline TurkishFilter/sections/specific.txt:396]: It seems that some ads on main page are not blocked.
<details><summary>Screenshot</summary>

image

</details>

Could you please block them too?

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:396]: i guess they changed something, omw

**Comment by stanislav-atr** [comment]
> Hey dude, I'm new here. How are you doing commit into "master"?

@gork7777 , hi! You are not supposed, actually. :) Do a pull request and wait for review.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -389,6 +389,12 @@ selamtuzla.com##.aside_ad
 kibriswebhaber.com##img[width="1100"][height="140"]
 petroturk.com##.site-content > div[class^="site-"]
 petroturk.com##div[class^="ads-"]
+unyenethaber.com##div[style^="width: 980px; height: auto; margin: 20px"]
+unyenethaber.com##.haberDetayIcerik > div[style^="margin-right: 10px; float: left;"]
+unyenethaber.com###siteSol > div[style^="width: 780px; height: auto;"]
+unyenethaber.com###siteSag > div[style^="width: 370px; height: auto;"]
+unyenethaber.com###haberAlani > div[style^="width: 100%; height: auto; overflow: hidden;"]
+unyenethaber.com###siteSol > div[style^="width: 728px; height: auto; margin: 20px"]
```
Comment (AdamWr) [inline TurkishFilter/sections/specific.txt:397]: It looks like that they changed something again and there is a different ad at the top.
<details><summary>Screenshot</summary>

image

</details>

There is video player on main page and this rule - `unyenethaber.com###siteSol > div[style^="width: 728px; height: auto; margin: 20px"]` hides it. I'm not sure if this video player should be blocked.

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:397]: > It looks like that they changed something again and there is a different ad at the top.
> 
Yeah, they've added `text-align: center` a few mins ago to `div[style]`

**Review by AdamWr — APPROVED** [review]

**Pull request merged**